ports/espressif/CMakeLists.txt

```diff
@@ -12,7 +12,11 @@ if("${CIRCUITPY_ESPCAMERA}")
 message("Including esp32-camera")
 set(EXTRA_COMPONENT_DIRS "esp32-camera")
 list(APPEND COMPONENTS "esp32-camera")
-message("COMPONENTS = ${COMPONENTS}")
+endif()
+
+if("${CIRCUITPY_DOTCLOCKFRAMEBUFFER}")
+message("Including esp32-camera")
+list(APPEND COMPONENTS "esp_lcd")
```

Comment:
Please always include it. In my IDF 5 update above I've always included camera. That ensures that the sdkconfig settings are always available. Later we can omit linking the library.

Reply:
OK.